src/utils/helpers.js

```diff
@@ -17,6 +17,26 @@ import { ValidationError } from '../utils/error';
 import commonOptions from '../utils/options';
 import { printResult } from '../utils/print';
 
+export const validatePublicKeys = publicKeys => publicKeys.map((publicKey) => {
+	try {
+		Buffer.from(publicKey, 'hex').toString('hex');
```

Comment:
`.toString('hex')` doesn't make sense unless you're checking if it's the same as the provided public key (which would be a good idea but would one of the checks below redundant).